src/shared/tpm2-util.h

```diff
@@ -50,6 +50,16 @@ static inline bool TPM2_PCR_MASK_VALID(uint32_t pcr_mask) {
 
 int dlopen_tpm2(void);
 
+/* Calls dlopen_tpm2() and returns 0 on success, < 0 on error. Also logs on error. Useful in "elvis operator"
+ * ternary checks. */
+static inline int tpm2_dlopen(void) {
+        int r = dlopen_tpm2();
```

Comment:
jmmpf. this is just very very confusing. the difference is so not obvious...

maybe instead just make the log level configurable, dlopen_many_sym_or_warn() accepts that too anyway